ambry-router/src/test/java/com/github/ambry/router/OperationTrackerTest.java

```diff
@@ -461,6 +461,49 @@ public void getOperationWithReplicaStateTest() {
     }
   }
 
+  @Test
+  public void noReplicaStateTtlUpdateTest() {
+    assumeTrue(operationTrackerType.equals(SIMPLE_OP_TRACKER));
+    List<Port> portList = Collections.singletonList(new Port(PORT, PortType.PLAINTEXT));
+    List<String> mountPaths = Collections.singletonList("mockMountPath");
+    // set up one node per data center for testing
+    datanodes = new ArrayList<>(Arrays.asList(new MockDataNodeId(portList, mountPaths, "dc-0"),
+        new MockDataNodeId(portList, mountPaths, "dc-1")));
+    mockPartition = new MockPartitionId();
+    localDcName = datanodes.get(0).getDatacenterName();
+    mockClusterMap = new MockClusterMap(false, datanodes, 1, Collections.singletonList(mockPartition), localDcName);
+    // put two STANDBY replicas in each data center (note that "populateReplicaList" method alternatively distributes
+    // the replica, so here we set 4 for two dc in total)
+    populateReplicaList(4, ReplicaState.STANDBY);
```

Comment:
Actually the test that I want you to check is for original dc, there's one replica down, one replica in non-state, and one can find the value. If you don't do the change, it will return unavailable. But with your change, it will return succeed since cross colo can do ttl update.

Reply:
Ok, updated.